automation/src/test/java/com/microsoft/identity/common/test/automation/AcquireTokenAuthorityMigrationTest.java

```diff
@@ -0,0 +1,157 @@
+package com.microsoft.identity.common.test.automation;
+
+import com.microsoft.identity.common.test.automation.actors.User;
+import com.microsoft.identity.common.test.automation.interactions.ClickDone;
+import com.microsoft.identity.common.test.automation.model.TokenCacheItemReadResult;
+import com.microsoft.identity.common.test.automation.questions.AccessToken;
+import com.microsoft.identity.common.test.automation.questions.ExpectedCacheItemCount;
+import com.microsoft.identity.common.test.automation.questions.TokenCacheItemCount;
+import com.microsoft.identity.common.test.automation.questions.TokenCacheItemFromResult;
+import com.microsoft.identity.common.test.automation.tasks.AcquireToken;
+import com.microsoft.identity.common.test.automation.tasks.AcquireTokenSilent;
+import com.microsoft.identity.common.test.automation.tasks.ExpireAccessToken;
+import com.microsoft.identity.common.test.automation.tasks.ReadCache;
+import com.microsoft.identity.common.test.automation.utility.Scenario;
+import com.microsoft.identity.common.test.automation.utility.TestConfigurationQuery;
+
+import net.serenitybdd.junit.runners.SerenityParameterizedRunner;
+import net.serenitybdd.screenplay.abilities.BrowseTheWeb;
+import net.thucydides.core.annotations.Managed;
+import net.thucydides.core.annotations.Steps;
+import net.thucydides.core.annotations.WithTag;
+import net.thucydides.junit.annotations.TestData;
+
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.openqa.selenium.WebDriver;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collection;
+
+import io.appium.java_client.service.local.AppiumDriverLocalService;
+
+import static net.serenitybdd.screenplay.GivenWhenThen.givenThat;
+import static net.serenitybdd.screenplay.GivenWhenThen.seeThat;
+import static net.serenitybdd.screenplay.GivenWhenThen.then;
+import static net.serenitybdd.screenplay.GivenWhenThen.when;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+
+@RunWith(SerenityParameterizedRunner.class)
+@WithTag("requires:none")
+public class AcquireTokenAuthorityMigrationTest {
```

Comment:
Let's change the name to be something more correct.  Authority Aliasing... we don't migrate anything.

Reply:
Thanks for the suggestion Shane. Renamed it.